.github/ISSUE_TEMPLATE/doc_request.yml

```diff
@@ -0,0 +1,40 @@
+name: Doc request
+description: Suggest doc changes for Pulsar
+title: "[Doc] "
+labels: ["doc-required"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please answer the following questions before submitting a doc issue. Thanks!
+  - type: textarea
+    attributes:
+      label: What issue do you find in Pulsar docs?
+      description: (For example, something missing, inaccurate, incomplete, hard to use/understand/find, etc)
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: What is your suggestion?
+      description: (For example, add explanations about xx, correct descriptions of xx, delete information about xx, etc)
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Do you have any references?
+      description: (For example, website links, etc)?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Would you like to fix this issue?
+      description: (Welcome to contribute docs! Pulsar grows from a tiny seed to a big tree with every little effort by us! Your contribution is the sunlight, water, and air!)
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Note
+      description: Please make some research on your question before submitting a doc issue. **Free rider is not welcomed**. Thanks!
+      options:
+        - label: I have made some research on my question and I need help on docs.
```

Comment:
What is the purpose of only having one option on a required list of check boxes? Don't we still want people to contribute doc requests even when they haven't been able to fully research the topic?

For example, if someone asks for a certain feature to be documented, and it is already documented, the underlying issue might be that the documentation isn't easily discoverable, which is still valuable to know.

Reply:
This is a soft reminder for issue raisers: please do research your issue and show us more context, so that we can understand your issue clearly and offers help quickly.

When triage issues in the community, you will find that some people just ask questions or make requests casually without reading existing docs carefully. They want to get the answers effortlessly and seems we owe them.

The open-source world welcomes anyone who respects others' work. Everyone needs to learn how to [ask the right questions](https://www.goodreads.com/book/show/394398.Asking_the_Right_Questions).